packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -336,15 +337,19 @@ const App = (props) => {
                                 <link
                                     rel="alternate"
                                     hrefLang={locale.id.toLowerCase()}
-                                    href={`${appOrigin}${buildUrl(location.pathname)}`}
+                                    href={`${appOrigin}${getPathWithLocale(locale.id, buildUrl, {
+                                        location
```

Comment:
If you pass the entire `location` object, the alternate link will also contain search params. I think ideally alternate link should point to the canonical URL, right?

```js
location: {
    pathname: location.pathname,
    search: ''
}
```

Or

```js
location: {
    ...location,
    search: ''
}
```
`getPathWithLocale` only uses `pathname` and `search`

Reply:
Thanks for catching that. I've made the change as suggested.